### PR TITLE
Dev ddeyc 73 - Youtube Video Embed

### PR DESCRIPTION
**Pull request by Paroxyse**
Add youtube video embed component to landing page

### CODE REVIEW

File: DD/Views/Partials/blocklistItems/_embededVideo.cshtml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <div class="embed-responsive embed-responsive-16by9 video-embed">
+    <iframe class="embed-responsive-item" src="@Model.VideoEmbedUrl.Url" title="YouTube video player" frameborder="0"
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+      referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+  </div>
+}
+else
+{
+
+}
```
Comment (rgr182) [inline DD/Views/Partials/blocklistItems/_embededVideo.cshtml:16]: No need for empty else

File: DD/Views/Partials/homePage/_imageCarousel.cshtml [hunk 2]
```diff
@@ -7,36 +7,46 @@
...
+    <link href="~/Styles/ContentCarousel.css" rel="stylesheet" />
 </head>
 
 <div class="container">
     <div class="row justify-content-center">
-        <div class="col-8"> 
+        <div class="col-8">
             <div id="imageCarousel" class="carousel slide" data-bs-ride="">
+                <ol class="carousel-indicators">
+                    @for (int i = 0; i < blocksList.Count; i++)
```
Comment (rgr182) [inline DD/Views/Partials/homePage/_imageCarousel.cshtml:18]: Try with for each instead if is not possible remember that arrays starts in 0 but count as validator can cause outbounds overflow exceptions.

File: DD/wwwroot/Styles/styles.css [hunk 3]
```diff
@@ -84,6 +84,61 @@ body {
 
 /* Add your custom styles here */
 
+/* Video embed */
+/* Styles for desktop */
```
Comment (rgr182) [inline DD/wwwroot/Styles/styles.css:88]: Duplicated comment

**Review by rgr182 — CHANGES_REQUESTED** [review]

File: DD/Views/Partials/homePage/_imageCarousel.cshtml (hunk 2 above)
Comment (rgr182) [reply DD/Views/Partials/homePage/_imageCarousel.cshtml:18]: Just check

File: DD/Views/Partials/blocklistItems/_embededVideo.cshtml (hunk 1 above)
Comment (Paroxyse) [reply DD/Views/Partials/blocklistItems/_embededVideo.cshtml:12]: Addressed in latest commit

**Review by rgr182 — APPROVED** [review]

**Pull request merged**